src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,51 @@
+.menu {
+display: flex;
+
+  &__list {
+    display: flex;
+    flex-direction: row;
+    margin: 0;
+    list-style-type: none;
+    padding: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $color-midnight-express;
+
+    transition-duration: 0.3s;
+    transition-property: all;
```

Comment:
It is recommended to specify the transition properties instead of 'all' for better performance